compiler/noirc_evaluator/src/ssa.rs

```diff
@@ -44,9 +44,16 @@ mod opt;
 pub(crate) mod parser;
 pub mod ssa_gen;
 
+#[derive(Debug, Clone)]
+pub enum SsaLogging {
+    None,
+    All,
+    Equals(String),
```

Comment:
```suggestion
    Contains(String),
```
I found it misleading, I thought in the end it will really look for equality.

Reply:
Oh, I changed between the behaviors a couple of times and I forgot to change this one.